products/cloudrun/terraform.yaml

```diff
@@ -98,9 +98,18 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           cloud_run_service_name: "cloudrun-srv"
         test_env_vars:
           project: :PROJECT_NAME
+    virtual_fields:
+      - !ruby/object:Provider::Terraform::VirtualFields
+        name: 'autogenerate_revision_name'
+        description: |
+          If set to `true`, the revision name (template.metadata.name) will be omitted and 
+          autogenerated by Cloud Run. This cannot be set to `true` while `template.metadata.name` 
+          is also set.
```

Comment:
It might be good to clarify somewhere that if `name` isn't in the config and this is false it will still be managed server side for legacy support.